src/main/java/app/pieceplayer/PiecePlayer.java

```diff
@@ -0,0 +1,11 @@
+package app.pieceplayer;
+
+import app.core.game.*;
+import app.core.interactor.*;
+
+/**
+ * Player wrapper that allows moving with pieces instead moves
+ * TODO: Implement methods that are not abstract
+ */
+public abstract class PiecePlayer<M extends Move<P>, P extends Piece> extends Player<M, P> {
+}
```

Comment:
I don't think putting `pieceplayer` package in the main directory is a good idea. It's a small utility package and should be treated as such, therefore consider moving it to `utils/pieceplayer` or something of that sort.